### ISSUE TITLE
Game crashes upon loading world :/

### ISSUE THREAD

**Issue by Hispanic-Productions** [simibubi/Above-and-Beyond#571]
I've been playing in a world for the past couple of days and everything was going fine, opening and closing didn't cause any problems. Suddenly, the same world I've been playing on is un-launchable and makes the game crash upon trying to open the world. After trouble shooting for quite some time, I decided to simply create a new world. I played on the new world for a couple of days, and suddenly the same problem, that makes the save un-launchable, happened to the save.

TL;DR: On two separate occasions, two saves crash the game upon opening either of them. Though, creating new saves does not result in the game crashing whenever opening them.

Any help would be very much appreciated.

Crash Report:
https://pastebin.com/1Fjijssz
